lib/access_import/acc_utils.rb

```diff
@@ -28,6 +28,14 @@ def parse_date_fields(record, *keys)
     record.merge(record.slice(*keys).compact.transform_values(&:to_date))
   end
 
+  def parse_boolean_fields(record, *keys)
+    record.merge(
+      parse_int_fields(record, *keys).slice(*keys).compact.transform_values do |val|
+        val == 1 ? true : false
```

Comment:
You could just do `val == 1` which already returns a boolean

Reply:
true